perfkitbenchmarker/linux_benchmarks/tensorflow_benchmark.py

```diff
@@ -97,6 +97,9 @@
 flags.DEFINE_boolean('tf_distortions', True,
                      '''Enable/disable distortions during image preprocessing.
                      These include bbox and color distortions.''')
+flags.DEFINE_string('tf_benchmarks_commit_hash',
+                    'bab8a61aaca3d2b94072ae2b87f0aafe1797b165',
+                    'git commit hash of desired tensorflow benchmark commit.')
```

Comment:
I'm not sure what's going on here with the diff. Odd merge or sync?

Reply:
Regarding the merge, I am unsure on this, and happy to re upload if required.

Reply:
Ok, let me know if you would like me to attempt this again